config/locales/defaults/ca.yml

```diff
@@ -15,7 +15,7 @@ ca:
     - dv
     - ds
     abbr_month_names:
-    -
+    - 
```

Comment:
Typo?

Reply:
> ferblape on 23 Jul Member
It looks like you added a space, but you can ignore it, we can't remove the blank here, it's a Rails default.

https://github.com/PopulateTools/gobierto/pull/3210/files?file-filters%5B%5D=.yml#r459266783